src/databricks/labs/lsql/dashboards.py

```diff
@@ -144,6 +149,170 @@ def from_path(cls, path: Path) -> "WidgetMetadata":
         return fallback_metadata.replace_from_arguments(shlex.split(first_comment))
 
 
+class Tile:
+    """A dashboard tile."""
+
+    def __init__(
+        self,
+        name: str,
+        content: str,
+        *,
+        position: Position = Position(0, 0, 0, 0),
+    ) -> None:
+        self._name = name
+        self._content = content
+        self.position = position
+
+        width, height = self._default_size
+        self.position = dataclasses.replace(position, width=position.width or width, height=position.height or height)
+
+    @property
```

Comment:
Don't use `property`

Reply:
@nfx : And the `size` on WidgetMetdata? Should that also not be a property? I recall the reasoning was that it is better to avoid in case of inherentence

Reply:
For consistency I remove the property there as well